### DISCUSSION TITLE
how to corelate metrics, Trace and logs using OpenTelemetry

### DISCUSSION THREAD

**Discussion by ajprakas** [open-telemetry/opentelemetry-java#2643] [Q&A]
Is there an easy way to corelate metrics, Trace and logs using OpenTelemetry?

**Comment by jkwatson** [comment]
At the moment, the metrics specification is not complete, and logging is even further down the road. So, currently, there isn't an easy way to do this. This is one of the long-term design goals of the project, though, so this will be important moving forward.
